examples/dt/uni07eta/control-plane.md

```diff
@@ -0,0 +1,56 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
+
+## Initialize
+
+Switch to the "openstack" namespace
+
+```bash
+oc project openstack
+```
+
+Change to the uni07eta directory
+
+```bash
+cd architecture/examples/dt/uni07eta
+```
+
+Edit [service-values.yaml](service-values.yaml) and
```

Comment:
the path to service-values.yaml is wrong (it needs the control-plane directory)
```
[fultonj@runcible uni07eta{uni-etha}]$ pwd
/home/fultonj/review/architecture/examples/dt/uni07eta
[fultonj@runcible uni07eta{uni-etha}]$  find . -name service-values.yaml
./control-plane/service-values.yaml
[fultonj@runcible uni07eta{uni-etha}]$ 
```

Reply:
Hello, this is a copy-paste of what we were told to use as a baseline – please ask the questions there: https://github.com/openstack-k8s-operators/architecture/pull/83

Reply:
resolved. Thanks!